pengwin-setup.d/cassandra.sh

```diff
@@ -43,18 +43,19 @@ EOF
     echo "%sudo   ALL=NOPASSWD: ${startCassandra}" | sudo EDITOR='tee -a' visudo --quiet --file=/etc/sudoers.d/start-cassandra
 
     local profile_mountproc="/etc/profile.d/mount-proc.sh"
-    sudo tee "${profile_mountproc}" << EOF
+    sudo tee "${profile_mountproc}" << 'EOF'
 #!/bin/bash
 
 # Check if we have Windows Path
 # fixed GH#308
-if [[ "$(df /proc | grep proc)x" == "x" ]]; then
-
-  sudo ${mountProc}
+if [[ "$(df /proc | grep proc | cut -c47-51)" != "/proc" ]]; then
```

Comment:
One thing, fix this to be compatible with sh instead of bash. Because sometimes the profile.d files are executed by sh, for example when the shell is zsh

Reply:
@crramirez In general i'm agreeing. I have tested this with zsh. Because i'm using /bin/bash as interpreter line, it looks like zsh calls this file with the bash as interpreter. AFAIK we preinstalling the bash in Pengwin and use zsh or similar additional.
But if you want, i'm trying to find another solution for making it fit for sh.

Reply:
If you define zsh as the default shell and launch pengwin. The files in /etc/profile.d will be executed by zsh. This is why to be at the safe side, make it compatible to sh. The easiest way is to change the shebang to /bin/sh, then shellcheck will tell you what to change. Here an example: https://github.com/WhitewaterFoundry/pengwin-base/blob/master/profile.d/update-motd.sh another https://github.com/WhitewaterFoundry/pengwin-base/blob/master/profile.d/00-pengwin.sh